src/main/java/com/softserve/academy/spaced/repetition/domain/BasePrice.java

```diff
@@ -0,0 +1,19 @@
+package com.softserve.academy.spaced.repetition.domain;
+
+import javax.persistence.Column;
+import javax.persistence.MappedSuperclass;
+
+@MappedSuperclass
+public class BasePrice {
```

Comment:
Please make it abstract to guarantee that no instance of this class can be created.